stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/FormElement.kt

```diff
@@ -14,6 +14,13 @@ interface FormElement {
     val identifier: IdentifierSpec
     val controller: Controller?
 
+    /**
+     * Whether the form element allows user interaction.
+     *
+     * This is used to determine if vertical mode needs to show the form screen.
+     */
+    val allowsUserInteraction: Boolean
```

Comment:
Would this be useful for the standard mode we have today?

Reply:
I can't think of a use case for it. Did you have something in mind?